src/libs/PolicyUtils.js

```diff
@@ -27,7 +27,7 @@ function hasPolicyError(policy) {
     if (!_.isEmpty(errors)) {
         return true;
     }
-    const hasFieldErrors = _.some(lodashGet(policy, 'errorFields', {}), !_.isEmpty);
```

Comment:
This didn't work, and I don't know why. See this [example](http://jsfiddle.net/oah7kpu9/).